pkg/command/output.go

```diff
@@ -0,0 +1,141 @@
+	o.Items = append(o.Items, ApplicationRow{
+		Name:        item.Name.String(),
+		DisplayName: item.Title(),
+		Scenarios:   item.ScenarioCount,
+
+		ApplicationItem: *item,
+	})
+	return nil
+}
+
+// ExperimentRow is a table row representation of an experiment.
+type ExperimentRow struct {
+	Name         string            `table:"name" csv:"name" json:"-"`
+	DisplayName  string            `table:"Name,custom" json:"-"`
+	Observations int64             `table:"observations,wide" csv:"observations" json:"-"`
+	Labels       map[string]string `table:"labels,labels" csv:"label_,labels,flatten" json:"-"`
+
+	experiments.ExperimentItem `table:"-" csv:"-"`
+}
+
+// ExperimentOutput wraps an experiment list for output.
+type ExperimentOutput struct {
+	Items []ExperimentRow `json:"items"`
+}
+
+// Add an experiment item to the output.
+func (o *ExperimentOutput) Add(item *experiments.ExperimentItem) error {
+	o.Items = append(o.Items, ExperimentRow{
+		Name:         item.Name.String(),
+		DisplayName:  item.DisplayName,
+		Observations: item.Observations,
+		Labels:       item.Labels,
+
+		ExperimentItem: *item,
+	})
+	return nil
+}
+
+// TrialRow is a table row representation of a trial.
+type TrialRow struct {
+	Experiment     string            `table:"experiment,custom" csv:"experiment" json:"-"`
+	Name           string            `table:"name" json:"-"`
+	Number         int64             `table:"number,custom" csv:"number" json:"-"`
+	Status         string            `table:"status" csv:"status" json:"-"`
+	Assignments    map[string]string `csv:"parameter_,flatten" json:"-"`
+	Values         map[string]string `csv:"metric_,flatten" json:"-"`
+	FailureReason  string            `table:"failure_reason,wide" csv:"failure_reason" json:"-"`
+	FailureMessage string            `table:"failure_message,wide" csv:"failure_message" json:"-"`
+	Labels         map[string]string `table:"labels,labels" csv:"label_,labels,flatten" json:"-"`
+
+	experiments.TrialItem `table:"-" csv:"-"`
+}
+
+// TrialOutput wraps a trial list for output.
+type TrialOutput struct {
+	Items []TrialRow `json:"items"`
+}
+
+// Add a trial item to the output.
+func (o *TrialOutput) Add(item *experiments.TrialItem) error {
+	var experiment string
+	if item.Experiment != nil {
+		experiment = item.Experiment.DisplayName
+	}
+
+	assignments := make(map[string]string, len(item.Assignments))
+	for i := range item.Assignments {
+		assignments[item.Assignments[i].ParameterName] = item.Assignments[i].Value.String()
+	}
+
+	values := make(map[string]string, len(item.Values))
+	for i := range item.Values {
+		values[item.Values[i].MetricName] = strconv.FormatFloat(item.Values[i].Value, 'f', -1, 64)
+	}
+
+	o.Items = append(o.Items, TrialRow{
+		Experiment:     experiment,
+		Name:           experiments.JoinTrialName(item.Experiment, item.Number),
+		Number:         item.Number,
+		Status:         cases.Title(language.English).String(string(item.Status)),
```

Comment:
`language.English` not sure I know of this package. interesting.

Reply:
I only know about it now because I was using `strings.Title` and that was deprecated in favor of this.